frontend/src/components/pages/CreateReview/CreateReviewPage.tsx

```diff
@@ -0,0 +1,276 @@
+import { AddIcon, ArrowBackIcon, DeleteIcon } from "@chakra-ui/icons";
+import {
+  Box,
+  Button,
+  ButtonGroup,
+  Heading,
+  IconButton,
+  Input,
+  Radio,
+  RadioGroup,
+  Stack,
+  Text,
+} from "@chakra-ui/react";
+import React, { useEffect, useState } from "react";
+
+import { Book } from "../../../types/BookTypes";
+// import { Option } from "../../../types/TagTypes";
+// import BookModal from "./BookModal";
+import DeleteModal from "./DeleteBookModal";
+import DeleteReviewModal from "./DeleteReviewModal";
+import PublishModal from "./PublishModal";
+import ReviewEditor from "./ReviewEditor";
+import SingleBook from "./SingleBook";
+
+const CreateReview = (): React.ReactElement => {
+  // const [tagsSelected, setTagsSelected] = useState<Option[]>([]);
```

Comment:
Are these commented lines necessary? If so, could you add a comment detailing why?